crates/sui-indexer/src/lib.rs

```diff
@@ -72,7 +72,7 @@ pub struct IndexerConfig {
     pub client_metric_port: u16,
     #[clap(long, default_value = "0.0.0.0", global = true)]
     pub rpc_server_url: String,
-    #[clap(long, default_value = "9000", global = true)]
+    #[clap(long, default_value = "3030", global = true)]
```

Comment:
lets keep the default so we don't surprise anyone? don't know if the PE team use the default in our deployment

Reply:
oh yea - committed this one by mistake!